ui/app/AppLayouts/Wallet/stores/RootStore.qml

```diff
@@ -22,7 +22,7 @@ QtObject {
     property var generatedAccounts: walletSectionAccounts.generated
     property var appSettings: localAppSettings
     property var accountSensitiveSettings: localAccountSensitiveSettings
-    property string locale: appSettings.locale
+    property string locale: !!appSettings.locale ? appSettings.locale : ""
```

Comment:
The real problem here, there's no such thing as `appSettings.locale` :)

It should read:
```suggestion
    property string locale: Qt.locale().name
```

Reply:
done